### DISCUSSION TITLE
[Bug]: Bad Playback speed selector formatting

### DISCUSSION THREAD

**Discussion by FusselTV** [blakeblackshear/frigate#12750] [Report a Bug] [answered]
### Describe the problem you are having

For some reason, the playback speed selector is not opening in the right direction.

![image](https://github.com/user-attachments/assets/95def445-ab39-449b-91d4-9e2a20010203)

### Steps to reproduce

iPad 12.9 with Safari. For some odd reason, I only have this problem with Safari. When using Brave on this tablet, it works as expected.

### Version

0.14.0-da913d8 RC-2

### Frigate config file

```yaml
not nessacceary
```

### Relevant log output

```shell
not nessacceary
```

### Operating system

Proxmox

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

not nessacceary

**Comment by hawkeye217** [comment]
I can’t reproduce this on an iPad running iOS 17.5.1. What iOS version are you using?

**Reply by FusselTV** [reply]
![WhatsApp Image 2024-08-04 at 13 45 50](https://github.com/user-attachments/assets/6c53c8e6-59c9-4b69-8dc9-95a7cc52bbf9)
![WhatsApp Image 2024-08-04 at 13 47 12](https://github.com/user-attachments/assets/22dec8df-268e-4475-82fb-708e40be8547)
![WhatsApp Image 2024-08-04 at 13 44 32](https://github.com/user-attachments/assets/9eadaaae-755a-4c6a-a31a-404c8eac4cb3)

**Reply by hawkeye217** [reply]
If you rotate the device between portrait and landscape modes, does the menu change position?

**Reply by FusselTV** [reply]
Nope, and now I get the problem: my iPad is using the portrait view in landscape. When I am in portrait, everything works as expected, but when I rotate to landscape, it stays in the view you saw in the screenshot.

**Reply by hawkeye217** [reply]
Thanks for the bug report. I can reproduce your issue in a simulator.

**Reply by hawkeye217** [reply, marked as answer]
Fixed in https://github.com/blakeblackshear/frigate/pull/12753

**Reply by FusselTV** [reply]
Thanks for the quick fix. Looking forward to the next release candidate!

**Reply by hawkeye217** [reply]
Happy to help. We're in a code freeze at the moment, so unless there are major bugs found, this fix will end up in the first maintenance release of Frigate 0.14.
